### PR TITLE
feat:added documentation

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/153262694-749686f7-6446-471d-aaf5-7bbc3edb9124.mp4

### CODE REVIEW

**Review by FarhanMobashir — COMMENTED** [review]
All good ✅

**Review by mohammed-sas — APPROVED** [review]

File: components/images/images.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Images</title>
+    <link rel="stylesheet" href="../../docs/components.css">
+    <link rel="stylesheet" href="../../landing page/style.css">
+    <link rel="stylesheet" href="../../docs/navigation.css">
+    <link rel="stylesheet" href="../../docs/sidebar.css">
+    <link rel="stylesheet" href="../../docs/documentation.css">
+    <link rel="stylesheet" href="images.css">
+    <script src="https://kit.fontawesome.com/17540dc721.js" crossorigin="anonymous"></script>>
+    <title>Cards</title>
```
Comment (ttahm3d) [inline components/images/images.html:17]: There are two title elements in this file.

**Review by ttahm3d — COMMENTED** [review]
Have left a comment. Please take a look.

File: components/images/images.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Images</title>
+    <link rel="stylesheet" href="../../docs/components.css">
+    <link rel="stylesheet" href="../../landing page/style.css">
+    <link rel="stylesheet" href="../../docs/navigation.css">
+    <link rel="stylesheet" href="../../docs/sidebar.css">
+    <link rel="stylesheet" href="../../docs/documentation.css">
```
Comment (MaruthiPothuganti) [inline components/images/images.html:14]: you can import these Styles inside image.css and use the styles from that file link.

**Review by MaruthiPothuganti — APPROVED** [review]
Changes that might be helpful are suggested. Everything else looks good to merge.

File: components/images/images.html (hunk 2 above)
Comment (stutisk) [reply components/images/images.html:14]: will do that

**Pull request merged**